### PR TITLE
Docs: add differences w/ bitcraze API to intro

### PR DESCRIPTION
**Pull request by jpreiss**
After we merged (most of) the Crazyswarm firmware fork into the official firmware, a lot of features that were once Crazyswarm-exclusive can now be done with the Bitcraze Python library. It is no longer obvious to users why they would choose Crazyswarm over the Bitcraze library, so we should discuss it in the docs.

This may be a little long-winded, also please double-check for accuracy.

### CODE REVIEW

**Comment by jpreiss** [comment]
One thing I left out is performance. Based on https://github.com/bitcraze/crazyflie-lib-python/issues/118 it seems like Crazyswarm can control more CFs per radio. Is this true?

File: docs/index.rst [hunk 1]
```diff
@@ -1,22 +1,58 @@
...
     </div>
 
+It is also possible to use the Crazyflie as a control board for other quadcopter hardware.
+
+
+How is Crazyswarm different from Bitcraze's `Crazyflie Python API <https://github.com/bitcraze/crazyflie-lib-python>`_?
+-----------------------------------------------------------------------------------------------------------------------
+
+Both can be used to control several Crazyflies from a Python script.
+Overall, Crazyswarm has more features but is more complicated.
```
Comment (whoenig) [inline docs/index.rst:26]: I am not sure if we have more features. It's a very different feature set. The Bitcraze API also now has `Swarm` support, and of course they support all firmware features, while we only support whatever seems useful for (our) research.

File: docs/index.rst [hunk 2]
```diff
@@ -1,22 +1,58 @@
...
+Here are some differences:
+
+- **Motion capture integration.**
+  Crazyswarm contains drivers for common motion capture systems.
+  The Bitcraze API can *send* position measurements to the Crazyflie,
+  but does not know how to *get* position measurements from mocap hardware.
+- **Identical motion capture markers.**
```
Comment (whoenig) [inline docs/index.rst:33]: Identical/single motion capture markers tracking?

**Review by whoenig — COMMENTED** [review]
Looks great - minor comments below.

Performance is hard to measure. It certainly got a lot better since then and I am also currently working on a native communication backend that the lib (and we) might both use in the future. I think only we support broadcast communication, which helps a lot in certain scenarios (especially when a mocap is involved.) For me the TLDR is that if you want ROS and/or motion capture you use the Crazyswarm, otherwise the official lib.

File: docs/index.rst (hunk 1 above)
Comment (jpreiss) [reply docs/index.rst]: line removed.

File: docs/index.rst (hunk 2 above)
Comment (jpreiss) [reply docs/index.rst]: Added to share bullet w/ duplicated arrangements.

**Pull request merged**